pipenv/project.py

```diff
@@ -574,32 +578,55 @@ def clear_pipfile_cache(self):
         _pipfile_cache.clear()
 
     def _parse_pipfile(self, contents):
+        toml_encoder = toml.TomlPreserveInlineDictEncoder()
+        toml_encoder.get_empty_table()
```

Comment:
I can't see what the second line is used for.
These lines should be moved to the except block where they are actually used.

Reply:
ah yeah i just left these here by accident